src/actions/definitionActions.js

```diff
@@ -72,3 +82,22 @@ export function revertDefinitionAction(definition, values, name) {
     return dispatch(actions.success({ componentsWithoutChanges }))
   }
 }
+
+export function browseDefinitionsAction(token, entity, name) {
+  return dispatch => {
+    const actions = asyncActions(name)
+    dispatch(actions.start())
+    return getLowScoreDefinitions(token, entity).then(
+      result => {
+        dispatch(actions.success({ add: result }))
+        const toAdd = map(result, component => EntitySpec.validateAndCreate(component.coordinates)).filter(e => e)
+        dispatch(uiBrowseUpdateList({ updateAll: toAdd }))
+        const chunks = chunk(map(toAdd, component => EntitySpec.fromCoordinates(component).toPath(), 100))
+        Promise.all(chunks.map(throat(10, chunk => dispatch(getDefinitionsAction(token, chunk)))))
+          .then(() => uiInfo(dispatch, 'All components have been loaded'))
```

Comment:
likely only need the error case to notify the user. In the normal case the background loading should be hidden from the user.

Reply:
That's right, i'm gonna correct this one